test/language/module-code/export-star-as-dflt.js

```diff
@@ -19,5 +19,10 @@ features: [export-star-as-namespace-from-module]
 ---*/
 
 export * as default from './export-star-as-dflt_FIXTURE.js';
-import ns from './export-star-as-dflt.js';
+import Self from './export-star-as-dflt.js';
+import { default as named } from './export-star-as-dflt.js';
+import * as ns from './export-star-as-dflt.js';
+
+assert.sameValue(Self.x, 1, 'Module was re-exported under the name `default`');
+assert.sameValue(named.x, 1, 'named binding was re-exported under the name `default`');
 assert.sameValue(ns.default.x, 1, 'namespace was re-exported under the name `default`');
```

Comment:
@syg I added these other cases as valid ways to verify the same export-star-from.

At the same time, I just noticed we don't have a simple coverage for `import { default as ...` and this is the first actual test verifying a positive importing of a default named binding.